### PR TITLE
Eula screen base

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

- add Eula Base Screen
-
-

#### Screenshots / Screen Recording (if applicable)

- 
Screenshot 2024-02-16 at 2 03 55 PM
Screenshot 2024-02-16 at 2 04 06 PM
Screenshot 2024-02-16 at 2 04 56 PM
Screenshot 2024-02-16 at 2 05 02 PM

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/screens/EulaScreen/EulaScreenBase.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            backgroundColor: theme.colors.background,
+        },
+        checkbox: {
+            paddingLeft: 0,
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'flex-start',
+        },
+        checkboxLabel: { flexGrow: 0 },
+    });
```
Comment (JeffGreiner-eaton) [inline login-workflow/src/screens/EulaScreen/EulaScreenBase.tsx:54]: you can add in somewhere here....

/**
 * Component that contains the eula screen....
 *
 * @param {EulaScreenProps} props - props of screen component....
 *
 * @category Component
 */

File: login-workflow/src/screens/EulaScreen/EulaScreenBase.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
+import React, { useCallback, useRef, useState } from 'react';
+import { EulaScreenProps } from './types';
+import {
+    ErrorManager,
+    WorkflowCard,
+    WorkflowCardActions,
+    WorkflowCardBody,
+    WorkflowCardHeader,
```
Comment (JeffGreiner-eaton) [inline login-workflow/src/screens/EulaScreen/EulaScreenBase.tsx:8]: can you add WorkflowCardHeader title to base component? License Agreement

**Comment by JeffGreiner-eaton** [comment]
I don't know why I can step through all 3 screens in example and step all the way back to landing page just fine but the second attempt its locked up. Does anyone else see this?

**Comment by ArshdeepSingh3-eaton** [comment]
yes it's locking up but we can look at those when we implement that functionality

File: login-workflow/src/screens/EulaScreen/EulaScreenBase.tsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+            <WorkflowCardHeader {...headerProps} />
+            {Object.keys(instructionsProps).length !== 0 && <WorkflowCardInstructions {...instructionsProps} />}
+            <WorkflowCardBody scrollable={false}>
+                <View style={defaultStyles.container}>
+                    {checkboxProps?.disabled ? (
+                        <View style={defaultStyles.retryContainer}>
+                            <TouchableOpacity style={defaultStyles.retryBody} onPress={onRefetch}>
+                                <Icon source={{ name: 'refresh' }} />
+                                <Text variant={'titleSmall'} style={defaultStyles.text}>
+                                    Retry
```
Comment (surajeaton) [inline login-workflow/src/screens/EulaScreen/EulaScreenBase.tsx:136]: Please add and use the translation keys for Retry.

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Comment by ArshdeepSingh3-eaton** [comment]
we can but with that i cannot see all the props for the component when a scroll over it 
i comes something like this
Screenshot 2024-02-21 at 3 40 56 PM
instead of 
Screenshot 2024-02-21 at 3 43 26 PM

**Comment by joebochill** [comment]
> we can but with that i cannot see all the props for the component when a scroll over it i comes something like this Screenshot 2024-02-21 at 3 40 56 PM 

^ This is correct and this is the standard we are settling on.


> instead of 
> Screenshot 2024-02-21 at 3 43 26 PM

^ This is not correct — the functional component takes a _single_ param called `props`, not multiple params. You'll notice none of the other component libraries we use (React Native Paper or MUI) do this either. 

We can go in and add links to the API docs (this is common practice) — right now they won't work because nothing is in the main branch.

**Review by JeffGreiner-eaton — APPROVED** [review]

**Comment by JeffGreiner-eaton** [comment]
just update the conflict

**Pull request merged**